src/components/meetupSection.js

```diff
@@ -57,83 +59,95 @@ const MeetupSection = () => (
             4px 4px 0 ${theme.colors.lightBlue},
             5px 5px 0 ${theme.colors.lightBlue}
           `,
-        }}
-      >
-        Community Meet‑ups
-      </Text>
+          }}
+        >
+          Community Meet‑ups
+        </Text>
+
+        <Text
+          as="h3"
+          fontSize={[2, 3]}
+          color="white"
+          fontWeight="heading"
+          mb={2}
+          sx={{ textTransform: "uppercase" }}
+        >
+          Past
+        </Text>
 
-      <Box
-        sx={{
-          display: "grid",
-          gridGap: "3rem 2rem",
-          gridTemplateColumns: [
-            "auto",
-            null,
-            `repeat(auto-fit, minmax(25em, 1fr))`,
-          ],
-        }}
-      >
-        {meetups.map(({ id, logo, title, date, venue, replay }) => (
+        {data.map(({ id, name, start_time, place }) => (
```

Comment:
Can you have a `map` for upcoming events too?

Reply:
I haven't tested it yet, I don't know what the data looks like for the upcoming events.

Reply:
Hmm, can you try creating an upcoming draft event (just don't publish it) so we can test it?

Reply:
We can test upcoming events if we have an active event.